core/app.go

```diff
@@ -50,28 +51,54 @@ func EmptyApp() *App {
 // LoadApp parses the commandline arguments and loads the config
 func LoadApp() (*App, error) {
 
-	var configFlag string
 	var versionFlag bool
-	var renderFlag string
 	var templateFlag bool
+	var reloadFlag bool
+
+	var configFlag string
+	var renderFlag string
+	var maintFlag string
+
+	var putMetricFlags MultiFlag
+	var putEnvFlags MultiFlag
 
 	if !flag.Parsed() {
-		flag.StringVar(&configFlag, "config", "",
-			"file path to JSON5 configuration file.")
+		flag.BoolVar(&versionFlag, "version", false,
+			"Show version identifier and quit.")
+
 		flag.BoolVar(&templateFlag, "template", false,
 			"Render template and quit. (default: false)")
+
+		flag.BoolVar(&reloadFlag, "reload", false,
+			"reload a ContainerPilot process through its control socket.")
+
+		flag.StringVar(&configFlag, "config", "",
+			"file path to JSON5 configuration file.")
+
 		flag.StringVar(&renderFlag, "out", "-",
 			"-(default) for stdout or file path where to save rendered JSON config file.")
-		flag.BoolVar(&versionFlag, "version", false, "Show version identifier and quit.")
+
+		flag.StringVar(&maintFlag, "maintenance", "",
+			"enable/disable maintanence mode through a ContainerPilot process control socket.")
```

Comment:
Super nitpick: I realize it's inconsistent in the existing, but can we be consistent with the config option descriptions in terms of first-letter capitalization? I don't have a strong opinion either way but we should pick one and stick to it.